app/views/case.py

```diff
@@ -245,7 +265,7 @@ def update_case():
             )
         )
 
-    preview_image_obj: FileStorage = form.preview_image.data
+    preview_image_obj: FileStorage = form.sub_title_image.data
```

Comment:
for me the "preview_image" is clearer and more understandable than the "sub_title_image",
the same with "sub_images"